Integrations/integration-QRadar.yml

```diff
@@ -408,5 +419,28 @@ script:
     - name: headers
       description: Table headers
     description: List all assets found in the model
+  - name: qradar-get-closing-reasons
+    arguments:
+    - name: include_reserved
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+      description: If true, reserved closing reasons are included in the response
+    - name: include_deleted
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+      description: If true, deleted closing reasons are included in the response
+    - name: fields
+      description: Use this parameter to specify which fields you would like to get
+        back in the response
+    - name: filter
+      description: This parameter is used to restrict the elements in a list base
+        on the contents of various fields
+    - name: Range
```

Comment:
Really needed ? Range?

Reply:
it is part of the api... not sure its needed